client/components/mma/billing/Billing.tsx

```diff
@@ -306,7 +305,29 @@ function renderProductBillingInfo([mmaCategory, productDetails]: [
 }
 
 function renderInAppPurchase(value: AppSubscription) {
-	return <div key={value.subscriptionId}>todo</div>;
+	return (
+		<div css={subHeadingBorderTopCss} key={value.subscriptionId}>
+			<h2
+				css={css`
+					${subHeadingTitleCss}
+					margin: 0;
+				`}
+			>
+				App Subscription
+			</h2>
+			<div
+				css={css`
+					${textSans.medium()};
+					border: 1px solid ${neutral[20]};
+					margin: ${space[5]}px 0;
+					padding: ${space[3]}px;
+				`}
+			>
+				To change your payment setup, please contact Apple (for iOS), or
```

Comment:
Don't we know whether the IAP is an Apple or Google one?

Reply:
Not that I can see - unless we can tell from the ProductID?
eg response
     ```
   {
            "subscriptionId": "",
            "from": "2023-01-11T11:05:20.000Z",
            "to": "2023-01-11T12:08:02.000Z",
            "valid": false,
            "gracePeriod": false,
            "autoRenewing": false,
            "productId": "uk.co.guardian.gla.1month.2018May.withFreeTrial"
        }
```

Reply:
Yes that product ID is the SKU basically, which is unique. Maybe we can have a mapping table / regex in MDAPI to make it useful for display @jonflynng ?

Reply:
These are the mappings: https://docs.google.com/spreadsheets/d/1I8XZnirRkIQ1RDAq9iSqVC5B9UzFqV4iJnC_dTfydfE/

Reply:
Will merge this PR, then discuss with Daniel B if there are other places we could use this info and raise a new PR